actuator/src/main/java/org/tron/core/actuator/ExchangeCreateActuator.java

```diff
@@ -59,14 +61,14 @@ public boolean execute(Object object) throws ContractExeException {
 
       accountCapsule.setBalance(newBalance);
 
-      if (Arrays.equals(firstTokenID, "_".getBytes())) {
+      if (Arrays.equals(firstTokenID, TRX_SYMBOL_BYTES)) {
```

Comment:
Naming TRX_SYMBOL_BYTES is not appropriate. Please use more direct words. For example underline or undercode.

Reply:
ok

Reply:
> Naming TRX_SYMBOL_BYTES is not appropriate. Please use more direct words. For example underline or undercode.

what is the meaning of "undercode"? Are you sure?  The character "_" stands for TRX when creating a transaction of ExchangeCreate. IMO, the TRX_SYMBOL_BYTES or TRX_SYMBOL is the direct word.

Reply:
agree!